components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -122,6 +123,10 @@ export const AppRoutes = () => {
         return <WhatsNew onClose={() => setWhatsNewShown(false)} />;
     }
 
+    if (location.pathname === "/linkedin" && search.get("code") && search.get("state")) {
+        return <LinkedInCallback />;
```

Comment:
I didn't see this when I tried

Reply:
This is supposed to happen in the LinkedIn pop-up, which for some reason no longer shows up. Investigating.

Reply:
Fixed.